### PR TITLE
Fix/share password layout

### PR DESCRIPTION
**Pull request by aboredvaro**
Fixed some design-to-code errors

### CODE REVIEW

**Comment by aboredvaro** [comment]
@CandelR I've resolved 2 conflicts, but check those just in case, since other conflicts were automatically resolved by GitHub. When ready, please merge to master 🚀

**Review by CandelR — COMMENTED** [review]
We cannot merge this to `master` until we release the email change feature, as this branch has been taken out of staging and is currently testing that feature.
Tomorrow after deployment I will merge this branch.

**Review by CandelR — APPROVED** [review]

**Pull request merged**
